### PR TITLE
Update phoenix_live_view to v0.13

### PR DESCRIPTION
**Pull request by grzuy**

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -57,7 +57,22 @@ Hooks.CommandInput = {
 
 let csrfToken = document.querySelector('meta[name=\'csrf-token\']').getAttribute('content');
 /* eslint-disable-next-line camelcase */
-let liveSocket = new LiveSocket('/live', Socket, {hooks: Hooks, params: {_csrf_token: csrfToken}});
+let liveSocket = new LiveSocket(
+  '/live',
+  Socket,
+  {
+    hooks: Hooks,
+    params: {_csrf_token: csrfToken},
```
[the remark this answers is not in the document]
Comment (eldano) [reply assets/js/app.js:65]: @grzuy now that this is multiline the esling-disable comment on line 59 needs to be changed to disable a block

**Review by eldano — APPROVED** [review]
Small comment to remove a warning, other than that LGTM

File: assets/js/app.js [hunk 2]
```diff
@@ -57,7 +57,20 @@ Hooks.CommandInput = {
...
+    metadata: {
+      keydown: (e, el) => {
+        return { code: e.code };
```
Comment (eldano) [inline assets/js/app.js:68]: It looks like we're already getting this value under the key "key", it's probably a default for this event so there's no need to send it manually

Comment (grzuy) [reply assets/js/app.js:68]: Great call - thanks!

**Comment by grzuy** [comment]
Making it draft so we wait to have feature test coverage for history before merging this in.

**Comment by grzuy** [comment]
Updated this branch to be built on top of `test_history` and changed PR base.

**Pull request merged**